h2/src/test/org/h2/test/TestAll.java

```diff
@@ -876,7 +882,58 @@ private void test() throws SQLException {
         afterTest();
     }
 
-    private void testUnit() {
+    /**
+     * Run additional tests.
```

Comment:
Rather just move these into test() inside a
  if (!networked)
block?

test() should also be renamed to testDb() probably

Reply:
This will increase testing time, because `testAdditonal()` is also skipped in some other modes, check for `!networked` is placed here just to prevent accidental incorrect invocation.

Actually we need to mark in each such test flags that are relevant to it (in annotations or in some method) and perform testing only with settings that have effect on such test, but this is much more complicated.

Reply:
`test()` and `testAdditional()` both use normal database connections. `test()` can be renamed into `testDb()`, but in this case `testAdditional()` should also be renamed to something like `testDbAdditional()` for consistency. Should I rename them or just leave them as is?

Reply:
Meh. Lets just land this as is. Possibly we can pull testUnit() and testAdditional() out into a separate runTest2() method and only test the desired combinations there.

We have too many possible combinations of flags to run each test with all possible combinations that could affect it. The best we can do is achieve reasonable coverage of combinations that all likely to surface bugs.